.storybook/preview-head.html

```diff
@@ -1,77 +1,91 @@
-<link rel="stylesheet" href="/node_modules/github-markdown-css/github-markdown.css">
-<link rel="stylesheet" href="/node_modules/highlight.js/styles/vs.css">
-<link rel="stylesheet" href="/src/styles/default-theme.css">
+<link rel="stylesheet" href="styles/github-markdown.css">
+<link rel="stylesheet" href="styles/vs.css">
+<link rel="stylesheet" href="styles/default-theme.css">
 <style>
+  html {
+    background-color: #f2f2f2;
+  }
 
-html {
-  background-color: #f2f2f2;
-}
+  body,
+  /* we need high specifity selectors to be able to override the autogenerated docs font settings */
+  .sbdocs *:where(:not(pre *)) {
+    font-family: "Liberation Sans", "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+  }
 
-body {
-  font-family: "Nunito Sans", -apple-system, ".SFNSText-Regular", "San Francisco", BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
-  overflow-y: scroll;
-  margin: 0;
-}
+  body {
+    overflow-y: scroll;
+    margin: 0;
+  }
 
-.story-shadow-container {
-  /* Sorry Storybook but our components aren't dark mode ready yet ;-) */
-  background-color: #fff;
-  box-shadow: 0 0 0.25em #ddd;
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  gap: 1em;
-  margin: 0 auto;
-  max-width: 72em;
-  padding: 1em;
-  width: 100%;
-}
+  /* we need high specifity selectors to be able to override the autogenerated docs font settings */
+  .sbdocs pre [class],
+  .sbdocs pre *,
+  .sbdocs code {
+    font-family: "Liberation Mono", monospace;
+  }
 
-/* Story viewer in docs mode */
-.docs-story .story-shadow-container {
-  padding: 0;
-  box-shadow: none;
-}
+  .sb-main-padded {
+    padding: 0 !important;
+  }
 
-.story-shadow-container[display-mode="flex-wrap"] {
-  flex-wrap: wrap;
-  flex-direction: row;
-}
+  .story-shadow-container {
+    /* Sorry Storybook but our components aren't dark mode ready yet ;-) */
+    background-color: #fff;
+    box-shadow: 0 0 0.25em #ddd;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+    margin: 0 auto;
+    max-width: 72em;
+    padding: 1em;
+    width: 100%;
+  }
 
-.title {
-  border-bottom: 1px solid #ccc;
-  color: #333;
-  font-family: "Nunito Sans", -apple-system, ".SFNSText-Regular", "San Francisco", BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
-  font-size: 13px;
-  font-weight: bold;
-  margin: 2rem -1rem 1rem;
-  padding-bottom: 0.25rem;
-  padding-left: 1rem;
-}
+  /* Story viewer in docs mode */
+  .docs-story .story-shadow-container {
+    box-shadow: none;
+  }
```

Comment:
Basically, we wanted to have some padding so that `beta` ribbons would overlap too much with the component in auto-generated docs.
Before (in prod)
![image](https://github.com/CleverCloud/clever-components/assets/100240294/18d9c383-5cb5-4bed-a28d-d1bc761ac056)
Now 
![image](https://github.com/CleverCloud/clever-components/assets/100240294/f8e86058-ee5b-4834-b033-ad3d7a96fd0b)